### ISSUE TITLE
Measure per-service test coverage for unit tests + VCR tests

### ISSUE THREAD

**Issue by melinath** [hashicorp/terraform-provider-google#18868]
### What kind of contribution is this issue about?

Pull request UX

### Details

We need to be able to measure test coverage on a per-service basis for PRs.

**Comment by shuyama1** [comment]
I'm a bit unclear about the test coverage for VCR tests in this context. Are we aiming to calculate field coverage in the acceptance tests? @melinath Could you please clarify the request? Thank you!

**Comment by melinath** [comment]
@shuyama1 no, this is just for standard code coverage - that is, how many LOC are exercised by the tests vs not. https://go.dev/blog/integration-test-coverage
